backend/unit_test.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+#---------------testing for database connection----------------- #
+include 'config.php';
+GLOBAL $db;
+print $db;
+
+#-----------------------account creation check ---------------------#
+include 'create.php';
+
+#case1: normal user
+$create_usr1 = array($name = "Bob", $password = "123abc");
+$_REQUEST['body'] = $json_encode($create_usr1);
+$_REQUEST['type'] = "create_account";
```

Comment:
after the account/tasks creation test, there is a portion where the tables are printed, any other case aside from the "normal" cases should not be present within the table(i.e missing/unexpected fields, as well as the empty case, which should not be allowed by the create functions

Reply:
This is a poor way to test because it is not automated enough (it requires the tester to read through what is printed and manually check if this is the expected output). I will approve for now, but the next pull request from you should be adding some conditional tests to this file